.github/workflows/cdh_basic.yml

```diff
@@ -45,12 +45,11 @@ jobs:
           fetch-depth: 1
 
       - name: Install Rust toolchain (${{ matrix.rust }})
-        uses: actions-rs/toolchain@v1
-        with:
-          profile: minimal
-          toolchain: ${{ matrix.rust }}
-          override: true
-          components: rustfmt, clippy
+        run: |
```

Comment:
Did you consider https://github.com/dtolnay/rust-toolchain for this as it seems widely used? This isn't nescessarily a suggestion, I've just realised that we use actions-rs all over the place and as you point out it has been unmaintained for a while and it using older node versions, so deciding on a single thing to replace it with (which could be just using cargo directly as you have), probably makes sense.

Reply:
I think it is a good choice. Prpbably we can replace all yamls with this GHA in another PR. But sure let me make an issue to record this

Reply:
finished in https://github.com/confidential-containers/guest-components/issues/675

Reply:
independent of this conversation, I had reported the same in https://github.com/confidential-containers/confidential-containers/issues/230. would be good to track the work for the whole project, not just guest-components?